### PR TITLE
Bikeshedding on top of #2870

### PR DESCRIPTION
**Pull request by rustyrussell**
See #2870

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,47 +52,69 @@ sudo apt-get install bitcoind lightningd
 
 ### Starting `lightningd`
 
-In order to start `lightningd` you will need to have a local `bitcoind` node running (in this case we start `testnet`):
+If you want to experiment with `lightningd`, there's a script to set
+up a `bitcoind` regtest test network of two local lighting nodes,
```
Comment (ZmnSCPxj) [inline README.md]: s/lighting/lightning/

Comment (rustyrussell) [reply README.md]: I need a better spellchecker!

**Comment by ZmnSCPxj** [comment]
ACK a92b8dd

**Comment by cdecker** [comment]
Rebased and squashed

ACK https://github.com/ElementsProject/lightning/pull/2932/commits/327be46abb5300ddf23a0b191fd5ec4c018acf58

**Pull request merged**